<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="24" passed="20">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="93432" started-at="2018-07-08T11:02:53Z" finished-at="2018-07-08T11:04:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="93432" started-at="2018-07-08T11:02:53Z" finished-at="2018-07-08T11:04:26Z">
      <class name="Admin_home.LoansFuction">
        <test-method status="PASS" signature="browserLunch()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="browserLunch" duration-ms="6139" started-at="2018-07-08T16:32:53Z" finished-at="2018-07-08T16:32:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserLunch -->
        <test-method status="PASS" signature="loginDetail()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="loginDetail" duration-ms="1848" started-at="2018-07-08T16:32:59Z" finished-at="2018-07-08T16:33:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDetail -->
        <test-method status="PASS" signature="checkAppDetail()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="checkAppDetail" duration-ms="733" started-at="2018-07-08T16:33:01Z" finished-at="2018-07-08T16:33:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkAppDetail -->
        <test-method status="PASS" signature="jumpToMemberProfile()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="jumpToMemberProfile" duration-ms="10088" started-at="2018-07-08T16:33:01Z" finished-at="2018-07-08T16:33:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jumpToMemberProfile -->
        <test-method status="PASS" signature="ChangePasswordForMember()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="ChangePasswordForMember" duration-ms="2033" started-at="2018-07-08T16:33:11Z" finished-at="2018-07-08T16:33:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChangePasswordForMember -->
        <test-method status="PASS" signature="backNevigationForMemberModify()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="backNevigationForMemberModify" duration-ms="10814" started-at="2018-07-08T16:33:13Z" finished-at="2018-07-08T16:33:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- backNevigationForMemberModify -->
        <test-method status="FAIL" signature="modifyMemberProfile()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="modifyMemberProfile" duration-ms="2015" started-at="2018-07-08T16:33:24Z" finished-at="2018-07-08T16:33:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [40] but found [86]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [40] but found [86]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Admin_home.MemberProfileModify.modifyMemberProfile(MemberProfileModify.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifyMemberProfile -->
        <test-method status="PASS" signature="historyTab()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="historyTab" duration-ms="62" started-at="2018-07-08T16:33:26Z" finished-at="2018-07-08T16:33:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- historyTab -->
        <test-method status="PASS" signature="getHistoryTable()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="getHistoryTable" duration-ms="2465" started-at="2018-07-08T16:33:26Z" finished-at="2018-07-08T16:33:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getHistoryTable -->
        <test-method status="PASS" signature="adModule()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="adModule" duration-ms="1848" started-at="2018-07-08T16:33:29Z" finished-at="2018-07-08T16:33:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adModule -->
        <test-method status="PASS" signature="modifyAdvertisment()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="modifyAdvertisment" duration-ms="13456" started-at="2018-07-08T16:33:31Z" finished-at="2018-07-08T16:33:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifyAdvertisment -->
        <test-method status="PASS" signature="removeAdvertisment()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="removeAdvertisment" duration-ms="1643" started-at="2018-07-08T16:33:44Z" finished-at="2018-07-08T16:33:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeAdvertisment -->
        <test-method status="FAIL" signature="addNewAd()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="addNewAd" duration-ms="4036" started-at="2018-07-08T16:33:46Z" finished-at="2018-07-08T16:33:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[the valu of title must take 4 to 25 but it take more then 25 expected [25] but found [45]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: the valu of title must take 4 to 25 but it take more then 25 expected [25] but found [45]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at Admin_home.AdvertisementsManage.addNewAd(AdvertisementsManage.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewAd -->
        <test-method status="FAIL" signature="disableExpire()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="disableExpire" duration-ms="37" started-at="2018-07-08T16:33:50Z" finished-at="2018-07-08T16:33:50Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: {"method":"id","selector":"newButton"}
Command duration or timeout: 36 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'IBM-Testing04', ip: '169.254.34.79', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=44.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 945b35be-8c8c-44a8-a8c6-0d9270a82f9e
*** Element info: {Using=id, value=newButton}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"newButton"}
Command duration or timeout: 36 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'IBM-Testing04', ip: '169.254.34.79', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=44.0, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 945b35be-8c8c-44a8-a8c6-0d9270a82f9e
*** Element info: {Using=id, value=newButton}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:413)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at Admin_home.AdvertisementsManage.disableExpire(AdvertisementsManage.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"id","selector":"newButton"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'IBM-Testing04', ip: '169.254.34.79', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/IBM/AppData/Local/Temp/anonymous3438777529645396992webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
	at <anonymous class>.FirefoxDriver.prototype.findElement(file:///C:/Users/IBM/AppData/Local/Temp/anonymous3438777529645396992webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10779)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/IBM/AppData/Local/Temp/anonymous3438777529645396992webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/IBM/AppData/Local/Temp/anonymous3438777529645396992webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12666)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/IBM/AppData/Local/Temp/anonymous3438777529645396992webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12608)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disableExpire -->
        <test-method status="PASS" signature="account()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="account" duration-ms="13213" started-at="2018-07-08T16:33:50Z" finished-at="2018-07-08T16:34:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- account -->
        <test-method status="FAIL" signature="clickPayment()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="clickPayment" duration-ms="2322" started-at="2018-07-08T16:34:03Z" finished-at="2018-07-08T16:34:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ it showing more then 'Debit to member' list  expected [Debit to member] but found [Community to member]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  it showing more then 'Debit to member' list  expected [Debit to member] but found [Community to member]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at Admin_home.AccountModule.clickPayment(AccountModule.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickPayment -->
        <test-method status="PASS" signature="afterPayType()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="afterPayType" duration-ms="486" started-at="2018-07-08T16:34:05Z" finished-at="2018-07-08T16:34:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterPayType -->
        <test-method status="PASS" signature="affterCommentClickToSubmit()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="affterCommentClickToSubmit" duration-ms="5910" started-at="2018-07-08T16:34:06Z" finished-at="2018-07-08T16:34:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- affterCommentClickToSubmit -->
        <test-method status="PASS" signature="accoutInfo()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="accoutInfo" duration-ms="1533" started-at="2018-07-08T16:34:12Z" finished-at="2018-07-08T16:34:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accoutInfo -->
        <test-method status="PASS" signature="accountName()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="accountName" duration-ms="2399" started-at="2018-07-08T16:34:13Z" finished-at="2018-07-08T16:34:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accountName -->
        <test-method status="PASS" signature="datePassing()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="datePassing" duration-ms="623" started-at="2018-07-08T16:34:16Z" finished-at="2018-07-08T16:34:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- datePassing -->
        <test-method status="PASS" signature="submit()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="submit" duration-ms="3131" started-at="2018-07-08T16:34:16Z" finished-at="2018-07-08T16:34:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submit -->
        <test-method status="PASS" signature="loanPageAfterClickSubmit()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="loanPageAfterClickSubmit" duration-ms="3409" started-at="2018-07-08T16:34:19Z" finished-at="2018-07-08T16:34:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loanPageAfterClickSubmit -->
        <test-method status="PASS" signature="viewLoan()[pri:0, instance:Admin_home.LoansFuction@c038203]" name="viewLoan" duration-ms="3130" started-at="2018-07-08T16:34:23Z" finished-at="2018-07-08T16:34:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- viewLoan -->
      </class> <!-- Admin_home.LoansFuction -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
